spec/controllers/users_controller_spec.rb

```diff
@@ -17,23 +17,25 @@
   end
 
   describe 'POST #create' do
-    after { ActionMailer::Base.deliveries.clear }
-
     context "with valid input" do
-      before do
+      it 'should create the user' do
```

Comment:
Don't recommend using 'should' in the spec message. Read more here -> http://betterspecs.org/#should